test/ocsp/helper/helper.go

```diff
@@ -326,11 +328,26 @@ func parseAndPrint(respBytes []byte, cert, issuer *x509.Certificate, config Conf
 	if resp.Certificate == nil {
 		fmt.Printf("  Certificate: nil\n")
 	} else {
-		fmt.Print("  Certificate:\n")
+		fmt.Printf("  Certificate:\n")
 		fmt.Printf("    Subject: %s\n", resp.Certificate.Subject)
 		fmt.Printf("    Issuer: %s\n", resp.Certificate.Issuer)
 		fmt.Printf("    NotBefore: %s\n", resp.Certificate.NotBefore)
 		fmt.Printf("    NotAfter: %s\n", resp.Certificate.NotAfter)
 	}
+
+	if errs != nil {
```

Comment:
You can do `if len(errs) > 0`, which IMO is more idiomatic in Go. It treats the nil case and the "non-nil but zero length" case the same. You can then simplify the loop below so it doesn't need to assign to `err`; it can simply return `errors[0]`.

Reply:
Good call. Still have to assign to err because I'm using it to accumulate all of the various errors in the slice (unless you're saying to accumulate directly into errs[0] instead of into a separate var?), but this does let me pull the initialization out of the loop and just iterate over the tail of the slice.